plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp

```diff
@@ -610,3 +615,32 @@ bool USentrySubsystem::IsPromotedBuildsOnlyEnabled()
 
 	return Settings->EnableForPromotedBuildsOnly;
 }
+
+void USentrySubsystem::ConfigureOutputDevice()
+{
+	OutputDevice = MakeShareable(new FSentryOutputDevice());
+	if (OutputDevice)
+	{
+		GLog->AddOutputDevice(OutputDevice.Get());
+		GLog->SerializeBacklog(OutputDevice.Get());
+	}
+}
+
+void USentrySubsystem::ConfigureOutputDeviceError()
+{
+	OutputDeviceError = MakeShareable(new FSentryOutputDeviceError(GError));
+	if (OutputDeviceError)
+	{
+		OutputDeviceError->OnError.AddLambda([this](const FString& Message)
+		{
+			SubsystemNativeImpl->CaptureAssertion(TEXT("Assertion failed"), Message);
```

Comment:
How do we know here, based on the checks, that this is an assertion and not a different kind of error?

Reply:
The `FSentryOutputDeviceError` makes the `FDebug::HasAsserted()` check when an error occurs and fires the `OnError` event only if it's true.